VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -92,12 +104,12 @@ public async Task<IActionResult> GetTestCallResultForParticipantAsync(Guid confe
 
         private async Task<Guid> GetIdForParticipantByUsernameInConference(Guid conferenceId, string username)
         {
-            var conference = _conferenceCache.GetConference(conferenceId);
+            var conference = await _conferenceCache.GetConferenceAsync(conferenceId);
             if (conference == null)
             {
                 var conferenceDetail = await _videoApiClient.GetConferenceDetailsByIdAsync(conferenceId);
-                await _conferenceCache.AddConferenceToCache(conferenceDetail);
-                conference = _conferenceCache.GetConference(conferenceId);
+                await _conferenceCache.AddConferenceAsync(conferenceDetail);
```

Comment:
would the new getOrAddToConference method be best suited here

Reply:
This should be a separate PR